store/sqlstore/user_store.go

```diff
@@ -1550,7 +1550,19 @@ func (us SqlUserStore) GetUsersBatchForIndexing(startTime, endTime int64, limit
 
 		var channelMembers []*model.ChannelMember
 		channelMembersQuery, args, _ := us.getQueryBuilder().
-			Select("cm.*").
+			Select(`
+				cm.ChannelId,
+				cm.UserId,
+				cm.Roles,
+				cm.LastViewedAt,
+				cm.MsgCount,
+				cm.MentionCount,
+				cm.NotifyProps,
+				cm.LastUpdateAt,
+				cm.SchemeUser,
+				cm.SchemeAdmin,
+				cm.SchemeGuest IS NOT NULL AND cm.SchemeGuest) as SchemeGuest
```

Comment:
This is because `SchemeGuest` can be null (because we don't migrate this data), and as NULL must be interpreted as False, to fit properly in the `model.ChannelMember` struct (if not it tries to assign `nil` to a `boolean`, and fails).

Reply:
Your parentheses in that query look unbalanced to me.

Reply:
Yep. Seems like a typo?

Reply:
Sorry, last minute reformat, I'll fix it

Reply:
Fixed, and good catch, thanks @reflog

Reply:
Multitasking is not my best skill, clearly. Thanks @grundleborg